app/views/header.scala.html

```diff
@@ -1,121 +1,150 @@
 @()(implicit request: Request[_])
 
-@activeLink(currentPath: String) = @{
-if(request.path.equals(currentPath)) "active"
-}
-
-<div class="wrapper">
-
-        <!-- Sidebar Holder -->
-    <nav class="navbar navbar-fixed-top" role="navigation" style="margin-bottom: 0">
-        <div class="navbar-header" style="height: 50px;">
-            <span class="main-logo" >
-                <a href="@routes.SessionsController.sessions(1, None)" class="pull-left logo">
-                    <img src='@routes.Assets.versioned("images/logo3.png")' style="padding-left: 15px;">
-                </a>
-            </span>
-        </div>
-        <ul class="nav navbar-nav navbar-right">
-        @if(SessionHelper.isLoggedIn) {
-            <li class="@activeLink(routes.UsersController.login().url) alignLinks" >
-                <a href="@routes.UsersController.login()"><i class="fa fa-sign-in"></i>
-                    <span>Login</span></a>
-            </li>
-            <li class="@activeLink(routes.UsersController.register().url) alignLinks"style="padding-right: 10px;">
-                <a href="@routes.UsersController.register()"><i class="fa fa-user-plus"></i>
-                    <span>Register</span></a>
-            </li>
-        } else {
-            <li class="dropdown alignLinks">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown" style="padding-right: 30px;">
-                    <i class="fa fa-user-circle-o"></i> <span>@SessionHelper.email.split("@")(0).split('.')(0)
+    @activeLink(currentPath: String) = @{
+        if(request.path.equals(currentPath)) "active"
+    }
+
+    @activeSidebar() = {
+        <script>
+                if (localStorage.hasActiveClass === "yes") {
+                    $('#sidebar').addClass("active");
+                } else {
+                    $('#sidebar').addClass("");
+                }
+        </script>
+    }
+
+    @activeCollapseButton() = {
+        <script>
```

Comment:
Can you add all scripting in js file instead of adding it directly to the template?